components/PopUp.js

```diff
@@ -47,7 +48,7 @@ export default class PopUp extends Component {
             this.toggleModal(true)
           }}
         >
-          <Icon name="ios-add" size={35} color="#f18d1a" />
+          <Icon name="ios-add" size={isTablet ? 60 : 35} color="#f18d1a" />
```

Comment:
I kinda dont like the approach of having to check isTablet on every component and set a different size. I think a better approach would be to apply a specific stylesheet for different devices (e.g. apply styles from tablet.js if it's a tablet)